charts/prow/templates/lego_config.yaml

```diff
@@ -0,0 +1,10 @@
+
+apiVersion: v1
+metadata:
+  name: '{{ template "fullname" . }}.lego'
+data:
+  # modify this to specify your address
+  lego.email: "{{ .Values.lego.email }}"
+  # configre letsencrypt's production api
+  lego.url: "https://acme-v01.api.letsencrypt.org/directory"
+kind: ConfigMap
```

Comment:
Probably one for a follow up PR, but could we switch to using helm dependencies to install kube-lego? That should give users customisation of a wider variety of kube-lego features without any more work on the test-infra side 😄